### ISSUE TITLE
Vaadin 19 projects break if you add a dependency when using npm

### ISSUE THREAD

**Issue by Artur-** [vaadin/flow#10306]
### Minimal reproducible example
```
vaadin init hello-vaadin --latest
cd hello-vaadin
# Add <pnpmEnable>false</pnpmEnable> to vaadin-maven-plugin in the production profile
mvn install -Pproduction
npm install open
node_modules/.bin/webpack
```

### Expected behavior
Webpack compiles the bundle

### Actual behavior
```
$ node_modules/.bin/webpack
/Users/artur/tmp/hello-vaadin/node_modules/webpack-cli/bin/cli.js:93
				throw err;
				^

Error: Cannot find module '@vaadin/stats-plugin'
```

### Reason

It seems that Flow messes with `node_modules/@vaadin` manually instead of using `npm`. 

`npm install open` cleans up the mess and logs 
```
removed 4 packages, and audited 1338 packages in 2s
```

### Versions:
    - Vaadin / Flow version: 19.0.0
    - npm: 7.6.0

**Comment by caalador** [comment]
Why would you manually build webpack after npm install instead of running the actual project?

**Comment by Legioth** [comment]
The whole point of Studio is to not have to run the actual project

**Comment by caalador** [comment]
Would need some thought and cleanup but https://github.com/vaadin/flow/tree/poc/install-plugins-npm would fix this issue.

**Comment by Artur-** [comment]
You also see this if you do `npm ls` in a project

```
npm ERR! code ELSPROBLEMS
npm ERR! extraneous: @vaadin/application-theme-plugin@0.3.3 /.../node_modules/@vaadin/application-theme-plugin
npm ERR! extraneous: @vaadin/stats-plugin@1.0.0 /.../node_modules/@vaadin/stats-plugin
npm ERR! extraneous: @vaadin/theme-live-reload-plugin@1.0.0 /.../node_modules/@vaadin/theme-live-reload-plugin
npm ERR! extraneous: @vaadin/theme-loader@0.0.4 /.../node_modules/@vaadin/theme-loader
```

**Comment by caalador** [comment]
The poc should also fix that.

**Comment by pleku** [comment]
While this is fixed by #10306 for now, not claiming that we have a feature for running webpack manually in the project.
